gax/src/main/java/com/google/api/gax/core/PropertiesProvider.java

```diff
@@ -54,6 +54,7 @@ public static String loadProperty(Class<?> loadedClass, String propertiesPath, S
       if (inputStream != null) {
         Properties properties = new Properties();
         properties.load(inputStream);
+        inputStream.close();
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#load(java.io.InputStream) can also throw exceptions, and you want `close()` to be called in that case.